.github/workflows/ci.yml

```diff
@@ -361,6 +379,9 @@ jobs:
             if [[ "${{ matrix.db }}" == "pgsql" ]] && ! [[ $GITHUB_REPOSITORY =~ /silverstripe-postgresql$ ]]; then
               composer require "silverstripe/postgresql:^2 || ^3" --no-update
             fi
+            if [[ "${{ matrix.db }}" == "sqlite3" ]] && ! [[ $GITHUB_REPOSITORY =~ /silverstripe-sqlite3$ ]]; then
```

Comment:
You'll notice that I'm not adding anything under the `services` key. That's because SQLite is already installed by default.

Reply:
Please add a comment to the services block at the top that sqllite3 is installed out of the box

Reply:
Added on line 264